src/app/shared/version/version.ts

```diff
@@ -0,0 +1,23 @@
+import {VERSION} from '@angular/material';
+
+/** This material version will be used in footer and stackblitz. */
+export const materialVersion = VERSION.full;
+
+/** Version information with title and redirect url */
+export interface VersionInfo {
+  url: string;
+  title: string;
+}
+
+/** Doc site versions. We update the urls and titles manually */
+// TODO(tinayuangao): Update the url with material.angular.io domain
+export const docVersions: VersionInfo[] = [
+  {
+    url: 'https://material2-docs-5.firebaseapp.com/',
+    title: '5.2.4'
+  },
+  {
+    url: `http://material2-docs-dev.firebaseapp.com`,
```

Comment:
is this supposed to be pointed at the dev one?

Reply:
No... I will update the url once we set up the domain for v5.